dashboard/src/components/dataBrowser/RuleGroupEditor/RuleEditorRow.tsx

```diff
@@ -0,0 +1,210 @@
+import ControlledSelect from '@/components/common/ControlledSelect';
+import ColumnAutocomplete from '@/components/dataBrowser/ColumnAutocomplete';
+import type { HasuraOperator } from '@/types/dataBrowser';
+import Option from '@/ui/v2/Option';
+import Text from '@/ui/v2/Text';
+import { useRouter } from 'next/router';
+import type { DetailedHTMLProps, HTMLProps } from 'react';
+import { useState } from 'react';
+import { useController, useFormContext } from 'react-hook-form';
+import { twMerge } from 'tailwind-merge';
+import RuleRemoveButton from './RuleRemoveButton';
+import RuleValueInput from './RuleValueInput';
+
+export interface RuleEditorRowProps
+  extends DetailedHTMLProps<HTMLProps<HTMLDivElement>, HTMLDivElement> {
+  /**
+   * Name of the parent group editor.
+   */
+  name: string;
+  /**
+   * Index of the rule.
+   */
+  index: number;
+  /**
+   * Function to be called when the remove button is clicked.
+   */
+  onRemove?: VoidFunction;
+  /**
+   * List of operators to be disabled for the rule editor.
+   *
+   * @default []
+   */
+  disabledOperators?: HasuraOperator[];
+}
+
+const commonOperators: {
+  value: HasuraOperator;
+  label?: string;
+  helperText?: string;
+}[] = [
+  { value: '_eq', helperText: 'is equal' },
+  { value: '_neq', helperText: 'is not equal' },
+  { value: '_in_hasura', label: '_in', helperText: 'is in (X-Hasura-)' },
+  { value: '_in', helperText: 'is in (array)' },
+  { value: '_nin_hasura', label: '_nin', helperText: 'is not in (X-Hasura-)' },
+  { value: '_nin', helperText: 'is not in (array)' },
+  { value: '_gt', helperText: 'is greater than' },
+  { value: '_lt', helperText: 'is lower than' },
+  { value: '_gte', helperText: 'is greater than or equal' },
+  { value: '_lte', helperText: 'is lower than or equal' },
+  { value: '_ceq', helperText: 'is equal to column' },
+  { value: '_cne', helperText: 'is not equal to column' },
+  { value: '_cgt', helperText: 'is greater than column' },
+  { value: '_clt', helperText: 'is lower than column' },
+  { value: '_cgte', helperText: 'is greater than or equal to column' },
+  { value: '_clte', helperText: 'is lower than or equal to column' },
+  { value: '_is_null', helperText: 'is null' },
+];
+
+const textSpecificOperators: typeof commonOperators = [
+  { value: '_like', helperText: 'is like' },
+  { value: '_nlike', helperText: 'is not like' },
+  { value: '_ilike', helperText: 'is like (case-insensitive)' },
+  { value: '_nilike', helperText: 'is not like (case-insensitive)' },
+  { value: '_similar', helperText: 'is similar' },
+  { value: '_nsimilar', helperText: 'is not similar' },
```

Comment:
I think we should remove "is". I understand the reasoning, but it looks too repetitive in the UI.